src/test/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/IntegerCopyNumberTransitionMatrixDataUnitTest.java

```diff
@@ -0,0 +1,131 @@
+
+    public static final RealMatrix HOMO_SAPIENS_COPY_NUMBER_TRANSITION_XX_X_TRUTH =
+            new Array2DRowRealMatrix(new double[][] {
+                    {0.99966751443861601, 0.0, 0.00033248556138398773, 0.0, 0.0},
+                    {0.0, 0.9997899779920747, 0.00021002200792527603, 0.0, 0.0},
+                    {4.6641935242897276e-08, 9.3423238818193651e-08, 0.99999985473174158, 5.0172599663674365e-09, 1.8582444319879394e-10},
+                    {0.0, 0.0, 4.541929579905158e-05, 0.99995458070420096, 0.0},
+                    {0.0, 0.0, 7.8833267638943636e-05, 0.0, 0.99992116673236109}}).transpose();
+
+    public static final RealMatrix HOMO_SAPIENS_COPY_NUMBER_TRANSITION_XX_Y_TRUTH =
+            new Array2DRowRealMatrix(new double[][] {{1.0}});
+
+    public static final RealMatrix HOMO_SAPIENS_COPY_NUMBER_TRANSITION_XY_X_TRUTH =
+            new Array2DRowRealMatrix(new double[][] {
+                    {0.99971173098797461, 0.00028826901202540391, 0.0, 0.0},
+                    {1.0067714836234777e-07, 0.99999989259309574, 6.5615120089096975e-09, 1.6824389766435122e-10},
+                    {0.0, 7.456796468461193e-05, 0.99992504963549644, 3.8239981889544576e-07},
+                    {0.0, 4.0420371867421184e-05, 8.0840743734842364e-06, 0.99995149555375906}
+            }).transpose();
+
+    public static final RealMatrix HOMO_SAPIENS_COPY_NUMBER_TRANSITION_XY_Y_TRUTH =
+            new Array2DRowRealMatrix(new double[][] {
+                    {0.99966851990709416, 0.00033148009290586881, 0.0, 0.0, 0.0},
+                    {5.9399783434370542e-08, 0.99999937404917871, 5.2251326738303193e-07, 3.4001255345191416e-08, 1.0036515132014333e-08},
+                    {0.0, 0.00016831138093714932, 0.99983149068329535, 1.9793576746822735e-07, 0.0},
+                    {0.0, 0.00035529148884256304, 6.4209305212511401e-06, 0.99963614727046246, 2.1403101737503797e-06},
+                    {0.0, 0.00027047913446676971, 0.0, 5.519982336056525e-06, 0.99972400088319713}
+            }).transpose();
+
+    @Test
+    public void testReadCopyNumberTransitionMatrixTable() {
+        final IntegerCopyNumberTransitionMatrixData loadedTransitionMatrix =
+                IntegerCopyNumberTransitionMatrixData.read(
+                        HOMO_SAPIENS_COPY_NUMBER_TRANSITION_AUTOSOMAL_TABLE_FILE, 0);
+        assertRealMatrixEquals(HOMO_SAPIENS_COPY_NUMBER_TRANSITION_AUTOSOMAL_TRUTH,
+                loadedTransitionMatrix.getTransitionMatrix(), 1e-16);
+    }
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testReadCopyNumberTransitionMatrixTableBadInput() {
+        IntegerCopyNumberTransitionMatrixData.read(HOMO_SAPIENS_COPY_NUMBER_TRANSITION_BAD_AUTOSOMAL_TABLE_FILE, 0);
+    }
+
+    public static void assertRealMatrixEquals(final RealMatrix expected, final RealMatrix actual, final double tol) {
```

Comment:
There's already an `assertEqualMAtrix` in `HDF5PCACoveragePoNUnitTest.java`.  It's worth keeping just one and putting it in a Utils class.